UM-USElab/gradecraft-development
# Fix radio button grading for pass/fail assignments

### Status
READY

### Description
Previously if the quick grade style for an assignment was set as radio and the assignment was of pass/fail type, then the instructor would be presented with a select list. Radio buttons would only appear with this option if the assignment also had levels.

This is a quick fix for the bug mentioned in #2956 and not a PR that addresses the need for a larger usability refactor.

### Gem dependencies
N/A

### Migrations
NO

### Steps to Test or Reproduce

1. Set the quick grade styling for a Pass/Fail type assignment to radio buttons.
2. Ensure that grades persist on save and that there are radio buttons for when there are and are not assignment levels.

### Impacted Areas in Application
Assignment mass_edit

======================
Closes #2956
